examples/linux_body_pose_estimation_imx477.py

```diff
@@ -0,0 +1,287 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
The date here should just be "2024".

Reply:
Hi,
I have made the necessary changes. Please review
Thanks